### DISCUSSION TITLE
[bug]  pure Edge Functions project = 'To save on cloud resources I just did a scan of all our projects and identified those which have been inactive for more than 7 days.' ?

### DISCUSSION THREAD

**Discussion by gowy222** [supabase/supabase#22351] [Questions]
I had a project only use Edge Functions as a small api backend, no-db/auth/storage used.
and the api used everyday (like 20times / hour).

but I got email: 
```
To save on cloud resources I just did a scan of all our projects and identified those which have been inactive for more than 7 days
... and it is scheduled to be paused in a couple of days

```
well, I dont wanna paused by supbase scan & identified sys in this case.
so, how supbase scan & identified projects?  Edge Functions usage is not count in 'active ' range?
little help pls?

**Comment by GaryAustin1** [comment]
Issue a request from your edge functions to the database or run a cron task to do a REST API call to the database API port.
